app/assets/stylesheets/admin/loans/_manage_loan_questions.scss

```diff
@@ -0,0 +1,31 @@
+// Manage Loan Questions
```

Comment:
Yes to your last question. Good ideas here. Will follow that logic for moving/renaming.